sympy/functions/elementary/piecewise.py

```diff
@@ -462,20 +465,22 @@ def _eval_subs(self, old, new):
     def _eval_transpose(self):
         return self.func(*[(e.transpose(), c) for e, c in self.args])
 
-    def _eval_template_is_attr(self, is_attr, when_multiple=None):
```

Comment:
`when_multiple` was used to make it emit something other than None when a multivalued expressions existed, e.g. `Piecewise((0, x< 0), (1, True)).is_zero` -> False but it should give None: the conditions do not allow us to say definitievely that the piecewise is zero. When we have `Piecewise((2, x< 0), (1, True)).is_zero` *then* we can say False.